app/controllers/api/authentication_controller.rb

```diff
@@ -0,0 +1,44 @@
+class Api::AuthenticationController < Api::ApplicationController
+  class AuthenticationError < StandardError; end
```

Comment:
this might be something you need on all controllers, not only in this controller